src/controllers/inbox.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import BaseAPIController from './BaseAPIController';
+import ImapProvider from '../providers/ImapProvider';
+import db from '../db';
+
+export class InboxController extends BaseAPIController {
+
+    /*Get INBOX data*/
+    getInbox = (req, res, next) => {
+        req.email.find().skip((req.params.page - 1) * 21).limit(21).exec(function(err, data) {
+            if (err) {
+                req.err = "invalid page"
```

Comment:
is this correct way to handle error? we faced problem in previous project because u were doing error like this